tests/NgTests/Infrastructure/MemoryStorage.cs

```diff
@@ -46,7 +46,8 @@ protected override Task OnSave(Uri resourceUri, StorageContent content, Cancella
         protected override Task<StorageContent> OnLoad(Uri resourceUri, CancellationToken cancellationToken)
         {
             StorageContent content;
-            Content.TryGetValue(resourceUri, out content);
+            // Ignore query strings
```

Comment:
That is an unnecessary assignment, I updated the comment to be more clearer. #wontfix